src/util.ts

```diff
@@ -20,7 +20,7 @@ export const PROVIDER_ERRORS = {
     jsonrpc: '2.0',
     id: IdGenerator(),
     error: {
-      code: -32005,
+      code: -32601,
```

Comment:
why `-32601`? I see it represents methodNotFound in https://github.com/MetaMask/rpc-errors/blob/main/src/error-constants.ts#L11. Based on the message passed after this line, `Limit exceeded`, I would think the original `-32005` would be expected.

could we use constants for these values?